prog/download_boot_image.rb

```diff
@@ -70,9 +74,13 @@ def blob_storage_client
         image_name: image_name,
         url: url,
         version: version,
-        sha256sum: sha256_sum
+        sha256sum: sha256_sum,
+        certs: download_from_blob_storage? ? Config.ubicloud_images_blob_storage_certs : nil
       }.to_json
-      sshable.cmd("common/bin/daemonizer 'host/bin/download-boot-image #{params_json.shellescape}' #{q_daemon_name}", stdin: Config.ubicloud_images_blob_storage_certs)
+      sshable.cmd(
+        "common/bin/daemonizer 'host/bin/download-boot-image' #{q_daemon_name}",
+        stdin: params_json
+      )
```

Comment:
nit: IMHO, it's not that long. We have the lines longer than it.
```suggestion
      sshable.cmd("common/bin/daemonizer 'host/bin/download-boot-image' #{q_daemon_name}", stdin: params_json)
```